### PR TITLE
Clarifying OSX, adding issue with Install-Package

### PR DESCRIPTION
**Pull request by halr9000**
- Changing first section to "non-Windows" and clarifying that most of these issues apply to both Linux & OSX.
- Fixed a few typos.
- Adding known issue with Install-Package per #1773

### CODE REVIEW

File: docs/KNOWNISSUES.md [hunk 1]
```diff
@@ -1,17 +1,22 @@
-Known Issues for PowerShell on Linux
+Known Issues for PowerShell on Non-Windows Platforms
 ====================================
 
-The first Alpha release of PowerShell on Linux is mostly functional but
+The first Alpha release of PowerShell on Linux and OSX is mostly functional but
```
Comment (TravisEz13) [inline docs/KNOWNISSUES.md]: We usually say `OS X` not `OSX'

File: docs/KNOWNISSUES.md [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
 does have some significant limitations and usability issues. 
 In some cases, these issues are simply bugs that haven't been fixed yet. 
 In other cases (as with the default aliases for ls, cp, etc.) we are
 looking for feedback from the community regarding the choices we make.
 
+Note: Due to the similarities of many underlying subsystems, PowerShell on Linux
+and Mac OSX tend to share the same level of maturity in both features and bugs.
```
Comment (TravisEz13) [inline docs/KNOWNISSUES.md]: We usually say `OS X` not `OSX`

File: docs/KNOWNISSUES.md [hunk 3]
```diff
@@ -45,14 +50,14 @@ preventing the script from working properly when interchanging objects.
 Missing command aliases
 -----------------------
 
-On Linux, the "convenience aliases" for the basic commands `ls`, `cp`,
+On Linux/Mac OSX, the "convenience aliases" for the basic commands `ls`, `cp`,
```
Comment (TravisEz13) [inline docs/KNOWNISSUES.md]: We usually say `OS X` not `OSX`

File: docs/KNOWNISSUES.md [hunk 4]
```diff
@@ -73,11 +78,11 @@ You can work around this by doing `ls (gci *.txt | % name)` or, more simply,
 .NET Framework vs .NET Core Framework
 -----------------
 
-PowerShell on Linux uses the .NET Core which is a subset of the full
+PowerShell on Linux/Mac OSX uses the .NET Core which is a subset of the full
```
Comment (TravisEz13) [inline docs/KNOWNISSUES.md]: We usually say `OS X` not `OSX`

File: docs/KNOWNISSUES.md [hunk 5]
```diff
@@ -101,21 +106,21 @@ Use `-Encoding ascii` to write ASCII text (which, not being Unicode, will not ha
 Job Control
 -----------
 
-There is no job-control support in PowerShell on Linux. 
+There is no job-control support in PowerShell on Linux/Mac OSX.
```
Comment (TravisEz13) [inline docs/KNOWNISSUES.md]: We usually say `OS X` not `OSX`

Comment (TravisEz13) [inline docs/KNOWNISSUES.md]: Please fix in all locations.

File: docs/KNOWNISSUES.md [hunk 6]
```diff
@@ -201,6 +206,13 @@ The following table lists commands that are known not to work in PowerShell on L
 </tr>
 </table>
 
+Installing Software using PackageManagement and PowerShellGet Modules
```
Comment (TravisEz13) [inline docs/KNOWNISSUES.md]: @bmanikm Has this been addressed in the latest builds?

Comment (bmanikm) [inline docs/KNOWNISSUES.md]: @TravisEz13 Yes, this issue got fixed in the latest builds. This is just a known issue for https://github.com/PowerShell/PowerShell/releases/tag/v6.0.0-alpha.9 release.

File: docs/KNOWNISSUES.md (hunk 5 above)
Comment (halr9000) [inline docs/KNOWNISSUES.md]: @TravisEz13 I just came across #1934, should I go with "macOS"?

**Comment by halr9000** [comment]
Better, @TravisEz13 ?

File: docs/KNOWNISSUES.md (hunk 6 above)
Comment (halr9000) [inline docs/KNOWNISSUES.md]: I added an explicit version note to address this.

File: docs/KNOWNISSUES.md [hunk 7]
```diff
@@ -201,6 +206,13 @@ The following table lists commands that are known not to work in PowerShell on L
...
+---------------------------------------
+- (v6.0.0-alpha.9) A bug in handling of System.Management.Automation.SemanticVersion as described in [#1618](https://github.com/PowerShell/PowerShell/issues/1618) prevents installing modules using 
+the Install-Module cmdlet due to the inability to parse the Alpha version string "6.0.0-alpha".
+This similarly affects the Install-Package cmdlet. A fix has been mergeed and will be in a future
```
Comment (TravisEz13) [inline docs/KNOWNISSUES.md]: `merged` is mispelled

**Comment by TravisEz13** [comment]
@halr9000 It's better.  I need to see if #1934 is the right decision, and probably a second reviewer.  Thanks.

File: docs/KNOWNISSUES.md (hunk 6 above)
Comment (andschwa) [inline docs/KNOWNISSUES.md]: Looks like we'll be doing an alpha.10 early next week then.

**Comment by andschwa** [comment]
LGTM

**Pull request merged**